### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by Sisyphus67**
Added explanation of and links to Turnkey Linux appliances

### CODE REVIEW

File: doc/installation.md [hunk 1]
```diff
@@ -10,8 +10,10 @@ If you are not confident installing Tracks from source using these instructions,
 
 * If you’d like to install Tracks on a local machine, try the packaged version offered by [Bitnami](http://bitnami.org/stack/tracks) — it runs on Windows, Mac OS X and Linux.
 * If you’d like to run Tracks within a dedicated virtual machine, [JumpBox](http://jumpbox.com/app/tracks) provides preconfigured virtual machine images.
+* If you'd like more installation opptions, Turnkey Linux offers a pre-built appliance available in many formats.   Supported formats are open virtual format (OVA, for VMWare or VirtualBox), bare metal or virtualization installation (ISO image), [Docker](https://hub.docker.com/r/turnkeylinux/tracks-14.1/), and deployment to Amazon EC2.  A full list of options can be seen on the [Turnkey Linux page](https://github.com/turnkeylinux-apps/tracks) and there is also a [GitHub project](https://github.com/turnkeylinux-apps/tracks).
```
Comment (mattr-) [inline doc/installation.md]: There's a typo here. Should be `options` instead of `opptions`

**Comment by mattr-** [comment]
Thanks for the update! Could make the one small change requested so that I can merge this in?

**Comment by dnrce** [comment]
I'm proposing #2082 instead of this change.

**Comment by dnrce** [comment]
I have added mention of Turnkey Linux to the wiki: https://github.com/TracksApp/tracks/wiki/Installation

@Sisyphus67, feel free to edit if needed.

Thanks!

**Pull request closed without merging**